test/WooPP_part1.test.ts

```diff
@@ -78,6 +80,7 @@ describe('WooPP Test Suite 1', () => {
     quoteToken = await deployMockContract(owner, IERC20.abi)
     baseToken1 = await deployMockContract(owner, IERC20.abi)
     baseToken2 = await deployMockContract(owner, IERC20.abi)
+    feeManager = await deployMockContract(owner, IWooFeeManager.abi)
```

Comment:
why we don't need to mock the methods of feeManager?

Reply:
Without trading, we don't need to mock feeManager.